### DISCUSSION TITLE
Can I move it to the Chinese version of Minecraft？

### DISCUSSION THREAD

**Discussion by chenguangsd** [Luke100000/minecraft-comes-alive#877] [Q&A]
My Chinese friends want to play this mod,so can I move it to the Chinese version of Minecraft?

**Comment by chenguangsd** [comment]
> My Chinese friends want to play this mod,so can I move it to the Chinese version of Minecraft?

of course it will be free to play.

**Comment by Luke100000** [comment]
Of course! Take a quick look at this repos licence. However, I thought Minecraft China Java has no mod loader support? How would you port?

**Reply by chenguangsd** [reply]
I'm sorry, but NetEase requires a clear commercial authorization certificate. Excuse me, but I'm sure this mod will be 100% free

**Reply by chenguangsd** [reply]
https://mcdev.webapp.163.com/#/component/edit/new this is the website.Please you think about it.I won't get any money from your mod.

**Reply by Luke100000** [reply]
This project including code and assets is licenced under the GPL 3.0, which allows commercial use as long as you licence and open source your project under the same licence. This licence is your authorization certificate.
https://github.com/Luke100000/minecraft-comes-alive?tab=GPL-3.0-1-ov-file#readme
